packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -9,50 +9,37 @@ versionLinks:
+Decorators are functions that can be used to declaratively annotate and modify the behavior of classes.
 
-Decorators are a [stage 3 proposal](https://github.com/tc39/proposal-decorators) for addition to the ECMAScript standard. Currently no browsers implement decorators, but compilers like [Babel](https://babeljs.io/) and [TypeScript](https://www.typescriptlang.org/) provide support for an earlier version of the decorators proposal. Lit decorators work with Babel and TypeScript, and will be updated to work with the final specification when it's implemented in browsers.
+Lit provides a set of optional decorators that enable declarative APIs for things like registering elements, defining reactive properties and query properties, or adding event options to event handler methods.
 
-See the [Enabling decorators](#enabling-decorators) section for more information.
-
-{% aside "info" %}
-
-What does stage 3 mean?
-
-Stage 3 means that the specification text is complete, and ready for browsers to implement. Once the specification has been implemented in multiple browsers, it can move to the final stage, stage 4, and be added to the ECMAScript standard. A stage 3 proposal can still change, but only if critical issues are discovered during implementation.
-
-{% endaside %}
-
-
-
-Lit supplies a set of decorators that reduce the amount of boilerplate code you need to write when defining a component. For example, the `@customElement` and `@property` decorators make a basic element definition more compact:
+For example, the `@customElement` and `@property()` decorators let you register a custom element and define a reactive property in a compact, declarative way:
 
 ```ts
 @customElement('my-element')
 export class MyElement extends LitElement {
-  @property() greeting = "Welcome";
-  @property() name = "Sally";
-  @property({type: Boolean}) emphatic = true;
-  //...
-}
-```
-{#custom-element}
 
-The `@customElement` decorator defines a custom element, equivalent to calling:
+  @property()
+  greeting = 'Welcome';
 
-```js
-customElements.define('my-element', MyElement);
+}
 ```
 
-The `@property` decorator declares a reactive property.
+{% aside "info" "no-header"%}
 
-See [Reactive properties](/docs/v3/components/properties/) for more information about configuring properties.
+Lit supports two different versions of the JavaScript decorators proposal – an early version supported by TypeScript that we refer to as _experimental decorators_ and a new and final version we refer to as _standard decorators_.
+ 
+There are some small differences in usage between the two proposals (standard decorators sometimes require the `accessor` keyword). Our code samples are written for experimental decorators because we recommend them for production at the moment.
```

Comment:
That sounds good to me. That's everything except `customElement` and `eventOptions` for our built-in decorators. Maybe we should put that info somewhere..

Reply:
Or we could just be specific here and say
> standard decorators on class fields require the `accessor` keyword

Reply:
Though we were trying to go for brevity here within the info box since more detail is linked to down below.

For some reason, I like the sound of "often" better than "usually".